src/utils/hints/stateChannelsHints.js

```diff
@@ -12,4 +12,30 @@ export const stateChannelsHints = {
   initiator: 'The account that initiated the creation of the state channel and covered the fees for its creation.',
   responder: 'The participating account that joined the state channel.',
   lastKnownRound: 'Participants can for example protect themselves from being cheated on by providing a snapshot from the latest off-chain round to the chain.',
+  slashedRound: 'The round in which the slash was triggered.',
+  slashedBy: 'The account that slashed (disputed) malicious ChannelCloseSoloTx or ChannelForceProgressTx.',
+  initiatorCreateAmount: 'Unsigned amount of coins the initiator commits to the channel.',
+  initiatorCloseAmount: 'Final balance for the initiator.',
```

Comment:
What is the difference between `initiatorCloseAmount` and `initiatorSettleAmount`? In what context are they being used?

Reply:
The context changes per tx type

SETTLE 
https://pr-331-aescan.stg.aepps.com/transactions/th_sf174Sa4cYhivSebQVt3UEvnzjXHg5o6Le3ZZkSu7fLcbpciu

vs 

CLOSE MUTUAL
https://pr-331-aescan.stg.aepps.com/transactions/th_2Fm4TFh1z8nMNhzBCLR62raVdnqKRjcQ2Gh8FH1fr2SUAyzUBB